### ISSUE TITLE
indiscriminate permissions for FileZilla

### ISSUE THREAD

**Issue by beroal** [roddhjav/apparmor.d#617]
In the `filezilla` profile, I see:

```
  / r,
  /*/ r,
  /*/*/ r,
```

This is a hell of a permission. I believe that FileZilla accesses these directories only because the embedded file manager shows them at the start.

Also I believe that

```
owner /dev/tty@{int} rw,
```

is not needed because FileZilla is GUI-only.

**Comment by roddhjav** [comment]
You are right, it is an issue. It should only have access to something like `@{user_sync_dirs}`. Regarding thee tty, that may be needed even for GUI app.

**Comment by roddhjav** [comment]
Meanwhile, they are required by default by FileZilla. So I don't thing it is a good idea to remove them.

**Comment by beroal** [comment]
> Meanwhile, they are required by default by FileZilla. So I don't thing it is a good idea to remove them.

I used FileZilla alright without
```
  / r,
  /*/ r,
  /*/*/ r,
  /dev/tty rw,
  owner /dev/tty@{int} rw,
```
The only problem is that FileZilla pollutes the log with its attempts to read the contents of every subdirectory of every directory I browse.

> You are right, it is an issue. It should only have access to something like `@{user_sync_dirs}`. Regarding thee tty, that may be needed even for GUI app.

I can't choose XDG directories here. I never cared about them. `download`, `projects` and `work` seem like good candidates too.
